schemas/context/profile-individual.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-individual",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Individual Profile  Mixin",
+  "type": "object",
+  "description": "This mixin groups a set of standard fields to describe the profile of an individual.",
```

Comment:
Desription conflicts with "context/profile" definition. Provide a Good description what this Mixin is supposed to be representing